spec/loadbalancer/README.md

```diff
@@ -1,14 +1,17 @@
-## EC Load Balancer Spec
+![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/enterpriseconnect/loadbalancer)
+## EC Gateway Load Balancer Spec
+### Goal
+High availability for EC gateway. Load balancing network traffic across gateway instances from multiple source and target agents in platform-agnostic way.
+
+### Runtime Requirement
+- VM's with same network group/ subnet. One VM for load balancer and rest for watchers
+- OS: Linux, Windows, Darwin, Arm, etc.
+- [docker 1x.xx+](https://docs.docker.com/get-docker/)   
+- [EC Agent](#tag-usage)
```

Comment:
what is tag usage? where is the compatibility? v1.1 vs v1.1beta, etc.

Reply:
Added compatability details in tabular format